src/app/components/MainLayout/style.ts

```diff
@@ -0,0 +1,25 @@
+import { makeStyles } from 'utils/styles';
+
+export const useStyles = makeStyles(theme => ({
+  root: {
+    background: theme.colors.codGray,
+    display: 'flex',
+    minHeight: '100vh',
+  },
+
+  headerAndContent: {
+    display: 'flex',
+    flex: 1,
+    flexDirection: 'column',
+    paddingLeft: '50px',
+    paddingRight: '30px',
+  },
+
+  header: {
+    paddingTop: '30px',
```

Comment:
mui has [these plugins](https://material-ui.com/styles/advanced/#jss-plugins) enabled and one of them jss-plugin-default-unit allows us to use just numbers like `paddingLeft: 50,`, px will be added in compiled css. Also possibly we should use theme.spacing everywhere to produce final value for a prop instead of hardcoded pixels

Reply:
Please consider using theme.spacing everywhere you've set px